src/metadata/Metadata.ts

```diff
@@ -193,11 +197,17 @@ export class Metadata extends MarketplaceApi {
      * @returns DDO of the asset.
      */
     public async retrieveDDO(
```

Comment:
this method should be made private to make sure internal parts of the sdk don't bypass resolve policies. For instance:  https://github.com/nevermined-io/sdk-js/blob/master/src/nevermined/Agreements.ts#L53